OasisDEX/oasis-borrow
# 🧭 Different liquidation pill color in Discover

# 🧭 Different liquidation pill color in Discover

Till liquidation <=10%: red
Till liquidation >10% && <=25%: orange
Till liquidation >25%: green
![image](https://user-images.githubusercontent.com/16230404/198308624-040b624e-42bd-4942-ad68-2fee9235ce29.png)
  
## Changes 👷‍♀️

- "Liquidated" and "Being liquidated" are always shown red as they are the worst status,
- "Till liquidation" color is displayed depending on proximity,
- removed `CLOSED_LONG_TIME_AGO` type from `DiscoverTableVaultStatus` enum since closed vault shouldn't be displayed.
  
## How to test 🧪

- See `/discover/highest-risk-positions` if "Till liquidation" pills colors are correct,
- you can modify `/public/mocks/discover/highest-risk-positions.json` file to see how it changes according to provided values.
